src/main/java/core/basesyntax/service/operation/impl/OperationPurchaseHandler.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.operation.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationHandler;
+
+public class OperationPurchaseHandler implements OperationHandler {
+    private static final String SEPARATOR_FOR_LINE_OPERATION_PRODUCTS_QUANTITY = ",";
+
+    @Override
+    public void operationHandler(FruitTransaction fruitTransaction) {
+        int quantityFromDb = operationDao.get(fruitTransaction.getFruit());
```

Comment:
operationDao is not defined in the class. You should either pass it as a constructor parameter or use dependency injection.